mmdeploy/codebase/mmpose/core/ops/adaptive_avg_pool_ncnn.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+
+class NcnnAdaptiveAvgPoolingOp(torch.autograd.Function):
```

Comment:
Why don't we add a symbolic rewriter in `mmdeploy/python`?

Reply:
> Why don't we add a symbolic rewriter in `mmdeploy/python`?

So what about `DetectionOutput` and `PriorBox` op in mmdet rewriter?

Reply:
adaptive_avg_pool is a buildin function in PyTorch, if we add symbolic in `mmdeploy/python`, we can reuse it in other repo.
`DetectionOutput` and `PriorBox` seen not that general.